wpilibjExamples/src/main/java/org/wpilib/examples/sysidroutine/SysIdRoutineBot.java

```diff
@@ -46,40 +45,40 @@ public void configureBindings() {
     // Using bumpers as a modifier and combining it with the buttons so that we can have both sets
     // of bindings at once
     m_driverController
-        .a()
-        .and(m_driverController.rightBumper())
+        .eastFace()
```

Comment:
```suggestion
        .southFace()
```
Matches the previous bindings and C++.